recipes/augustus/3.2.2/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -x -e
+
+export CC=${PREFIX}/bin/gcc
+export CXX=${PREFIX}/bin/g++
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export BOOST_INCLUDE_DIR=${PREFIX}/include
+export BOOST_LIBRARY_DIR=${PREFIX}/lib
+
+export CXXFLAGS="-DUSE_BOOST -I${BOOST_INCLUDE_DIR} -L${BOOST_LIBRARY_DIR}"
+export LDFLAGS="-L${BOOST_LIBRARY_DIR}"
+
+
+make
+mkdir -p $PREFIX/bin
+mkdir -p $PREFIX/scripts
+mkdir -p $PREFIX/config
+mv bin/* $PREFIX/bin/
+mv scripts/* $PREFIX/scripts/
+mv config/* $PREFIX/config/
+
+#Install perl reqs
+cpanm Cwd File::Basename FileHandle File::Path File::Spec::Functions Getopt::Long IO::File List::Util Term::ANSIColor
```

Comment:
@bgruening 

The conda skeleton cpan has a recursive option, and I'm working on a quick bootstrap to use that to add the perl modules, but I'm getting an error with Test::Deep. I've reported the error to conda build here:  [https://github.com/conda/conda-build/issues/994](conda-build 994)